app/controllers/solutions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SolutionsController < ApplicationController
+
+  def create
+    @solution = Solution.new(solution_params)
+    @solutions = @question.solutions
+    if @solution.save
+      redirect_to question_path(@question.id), notice: "回答の投稿をしました"
+    else
+      @solutions = Solution.all.order("id DESC")
```

Comment:
これでは，全ての回答が表示されてしまいます。
```suggestion
      @solutions = @question.solutions.order("id DESC")
```